### PR TITLE
Updated OS X install instructions

### PR DESCRIPTION
**Pull request by tbarsballe**
Reformatted to match other instructions. 
Added instructions for minor update, upgrade, uninstall. Added components page

### CODE REVIEW

File: docs/usermanual/source/intro/installation/mac/components.rst [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+   * - LIBTIFF
+     - TIFF Library and Utilities.
+   * - GEOTIFF
+     - LIBTIFF extension for reading and writing GeoTIFF information tags.
+   * - LASZIP
+     - Free and lossless LiDAR compression library.
+   * - PDAL
+     - Point Data Abstraction Library.
+   * - ANT
+     - Apache Ant build tool, used by Boundless SDK.
```
Comment (tbarsballe) [inline docs/usermanual/source/intro/installation/mac/components.rst]: This list is a bit more extensive than the one in the Windows components list, but exactly matches the applications listed by the CLI installer.

File: docs/usermanual/source/intro/installation/windows/minorupdate.rst [hunk 2]
```diff
@@ -24,7 +24,7 @@ Installation
 
       Starting and stopping OpenGeo Suite services
 
-Services can also be controlled from the Windows :guilabel:`Services` dialog available by navigating to :menuselection:`Administrative Tools --> Services` from the Windows :guilabel:`Control Panel`.
+   Services can also be controlled from the Windows :guilabel:`Services` dialog available by navigating to :menuselection:`Administrative Tools --> Services` from the Windows :guilabel:`Control Panel`.
```
Comment (tbarsballe) [inline docs/usermanual/source/intro/installation/windows/minorupdate.rst]: Fix numbering

File: docs/usermanual/source/intro/installation/mac/components.rst [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+      OpenGeo Suite Components
+
+.. only:: enterprise
+
+   .. figure:: img/apps-ee.png
+
+      OpenGeo Suite Components
+
+Geoserver
```
Comment (bmmpxf) [inline docs/usermanual/source/intro/installation/mac/components.rst]: GeoServer (sorry)

File: docs/usermanual/source/intro/installation/mac/components.rst (hunk 1 above)
Comment (bmmpxf) [inline docs/usermanual/source/intro/installation/mac/components.rst]: That's fine.

File: docs/usermanual/source/intro/installation/mac/include/componentinstall.txt [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+   #. Open **System Preferences**
+
+   #. Select **Security & Privacy**
+
+   #. Under **Allow apps downloaded from:** select **Anywhere**
+
+.. note:: The GeoServer application runs a Jetty web server that also contains GeoWebCache, GeoExplorer, and the OpenGeo Dashboard.
+
+You can check the server status and perform other tasks from the GeoServer icon in the OSX menu bar.
```
Comment (bmmpxf) [inline docs/usermanual/source/intro/installation/mac/include/componentinstall.txt]: OS X

File: docs/usermanual/source/intro/installation/mac/uninstall.rst [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+#. Shutdown GeoServer and PostGIS by selecting :guilabel:`Quit` from the GeoServer and PostGIS icons in the OS X menu bar.
+
+   .. figure:: img/ext_quit.png
+
+      Shutting down GeoServer
+
+#. Download the free AppCleaner utility from `FreeMacSoft <http://www.freemacsoft.net/appcleaner/>`_. Extract the file and run it.
+
+#. Drag the :guilabel:`Geoserver`, :guilabel:`Geoserver Extensions`, :guilabel:`PostGIS` and :guilabel:`PostGIS Utilities` applications into the AppCleaner window. Click the :guilabel:`Delete` button.
```
Comment (bmmpxf) [inline docs/usermanual/source/intro/installation/mac/uninstall.rst]: GeoServer (sorry again)

**Comment by bmmpxf** [comment]
Looks great, thanks.

**Pull request merged**